src/components/product-card/product-card.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container py-5">
+  <div>
+    <div class="col-md-8 col-lg-6 col-xl-4">
+      <div class="card">
+        <img src="{{imageURL}}"
+          class="card-img-top" alt="spongebob chair" />
```

Comment:
Alt text should probably also be a parameter. BUT we can wait to switch that over once we have a product interface to work with.